content/docs/ops/maintenance-list.md

```diff
@@ -61,7 +86,9 @@ Use the [AWS Console](http://docs.aws.amazon.com/govcloud-us/latest/UserGuide/go
 - RevokeSecurityGroupIngress
 - UpdateTrail
 
-If any suspicious activity is discovered discuss with appropriate team members and if necessary follow the [Security Incident Response Guide]({{< relref "docs/ops/security-ir.md" >}}).
+If you observe any non-Terraform activity, discuss the event(s) with the 
+indicated cloud.gov operator(s) and if necessary follow the 
```

Comment:
Can we make `indicated cloud.gov operator(s)` a link to [this Google doc](https://docs.google.com/spreadsheets/d/1mW3tphZ98ExmMxLHPogSpTq8DzYr5Oh8_SHnOTvjRWM/edit#gid=0)?